src/easylink/pipeline_schema_constants/testing.py

```diff
@@ -380,29 +380,8 @@
                     ),
                 ],
             ),
-            input_slots=[
-                InputSlot(
-                    name="step_1_main_input",
-                    env_var="DUMMY_CONTAINER_MAIN_INPUT_FILE_PATHS",
-                    validator=validate_input_file_dummy,
-                    splitter=split_data_in_two,
-                ),
-            ],
-            output_slots=[OutputSlot("step_1_main_output", aggregator=concatenate_datasets)],
```

Comment:
Is this (leaving the input and output slots implicit, and "inheriting" them from the child) consistent with TemplatedSteps?

Reply:
[yes](https://github.com/ihmeuw/easylink/blob/main/src/easylink/step.py#L708-L709). Though I'm rather surprised that TemplatedSteps don't require a deepcopy the way I definitely need to here.

Reply:
⬆️  nm, I'm dumb - TemplatedSteps aren't then proceeding to _modify_ the copied slots like EmbarrassinglyParallelSteps are

Reply:
I mean, it's still confusing to think about the parent and child having the exact same slot object, but definitely not something to change in this PR!